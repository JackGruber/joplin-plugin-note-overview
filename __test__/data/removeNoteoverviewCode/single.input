Some note body

<!-- note-overview-plugin
search: type:todo iscompleted:0 notebook:"3 - WAITING" tag:pro
fields: title, created_time, body, todo_due, tags
sort: created_time ASC
-->

and more Text