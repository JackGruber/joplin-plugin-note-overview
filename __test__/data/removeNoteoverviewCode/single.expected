Some note body



and more Text